nequip/train/trainer.py

```diff
@@ -721,6 +721,13 @@ def init(self):
             return
 
         self.model.to(self.torch_device)
+
+        self.rescale_layers = []
```

Comment:
none of the code is gonna try to pickle this array, right?

Reply:
no. since it is not stated as an init argument.

Reply:
:+1: